### PR TITLE
Port notes commands

### LINKED ISSUE #232: Port notes commands to v13
## Purpose
Port `cc!notes`, `cc!addnote`, `cc!removenote` to Discord.JS v13

### Is your feature request related to a problem? Please describe.
Currently all our commands (bar a handful like `cc!ping`) require porting over to Discord.JS v13 and the notes commands seemed like a good (albeit artibirtary) place to start. 

Most of the issues around v12 -> v13 are caused by embeds which looks to be a fairly minor change.

## Acceptance Criteria
Please list the requirements the implementation of this feature should meet.
- [ ] Aforementioned commands should work on Discord.JS v13.

### PR DESCRIPTION
**Pull request by lyallstewart**
## What issue is this solving?
Closes #232 

### Description
Ports the three notes commands over to v13.

## Any helpful knowledge/context for the reviewer?

- Is a re-seeding of the database necessary? ❌ 
- Any new dependencies to install? ❌ 
- Any special requirements to test? ❌ 
  - (e.g., admin perms, alt account, etc.)

### Please make sure you've attempted to meet the following coding standards
- [X] Code has been tested and does not produce errors
- [X] Code is readable and formatted
- [X] There isn't any unnecessary commented-out code

### CODE REVIEW

**Comment by lyallstewart** [comment]
Completely managed to screw up the Git log there.... I think I'll just avoid rebasing from now on....

**Comment by lyallstewart** [comment]
There we go!

File: config/filters.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ const map = new Map([
   ['$', 's'],
   ['0', 'o'],
   ['7', 't'],
-  ['3', 'e']
+  ['3', 'e'],
```
Comment (lyallstewart) [inline config/filters.js:17]: One of the formatting extensions, not a deliberate change

File: commands/moderation/notes.js [hunk 2]
```diff
@@ -62,11 +62,11 @@ function notesLog(msg, targetUser, notes) {
       )
       .setColor(embedFlair[Math.floor(Math.random() * embedFlair.length)])
       .addField(`Total`, `${totalNotes}`)
-      .addField(`Latest Notes: `, listOfNotes)
+      .addField(`Latest Notes: `, listOfNotes.join('\n'))
```
Comment (aedwardg) [inline commands/moderation/notes.js:65]: This is interesting, did they change it so you can't pass in a list here anymore? That's good to know.

**Review by aedwardg — APPROVED** [review]
Looks good 👍

**Review by mikejoh12 — APPROVED** [review]
Great work!

**Pull request merged**